src/Core/Utilities/CoreHelpers.cs

```diff
@@ -242,6 +242,17 @@ public async static Task<X509Certificate2> GetEmbeddedCertificateAsync(string fi
             }
         }
 
+        public static string GetEmbeddedResourceContentsAsync(string file)
```

Comment:
Is this used anywhere?

Reply:
Yes, I'm using it to load the migration scripts into EF for MySQL and Postgres migrations. Otherwise, we'd have to have those terrible SQL scripts in a C# file